plugins/che-plugin-analytics/che-plugin-analytics-wsagent/src/main/java/com/redhat/che/plugin/analytics/wsagent/AnalyticsEvent.java

```diff
@@ -19,7 +19,9 @@
   WORKSPACE_USED("Use Workspace in Che"),
   WORKSPACE_INACTIVE("Keep Workspace Inactive in Che"),
   WORKSPACE_STOPPED("Stop Workspace in Che"),
-  EDITOR_USED("Edit Workspace File In Che", 30, new String[] {PROGRAMMING_LANGUAGE});
+  EDITOR_USED("Edit Workspace File In Che", 30, new String[] {PROGRAMMING_LANGUAGE}),
+  PUSH_TO_REMOTE("Push to remote"),
+  COMMIT_LOCALLY("Commit locally");
```

Comment:
Also could you add `in Che` at the end of the event names please ?

This makes it very easy to group all the events that happen in a Che workspace, which is necessary to sum-up the whole usage duration (since there's only 1 current action at a given time).

Reply:
Done.